Packages/Models/Sources/Models/Status.swift

```diff
@@ -71,7 +72,11 @@ protocol StatusUI {
 }
 
 public final class Status: AnyStatus, Codable, Identifiable, Equatable, Hashable, StatusUI {
-  public var userMentioned: Bool?
+  private let _userMentioned: UnsafeAtomic<Bool>
```

Comment:
I'm wondering if this could leave somewhere else. It could probably be a purely view and calculated in the init of the row view model. So we don't have to male the sacrifice of making this a custom decoder for this class.

Reply:
Maybe it could be bested in a sub structure? So we would decode only this prop manually?